readthedocs/core/views/__init__.py

```diff
@@ -50,33 +63,20 @@ class HomepageView(TemplateView):
       2. when user is logged off, redirect to login page
     """
 
-    template_name = "homepage.html"
-
     def get(self, request, *args, **kwargs):
-        # Request hitting the old domain (readthedocs.org / readthedocs.com)
-        if request.get_host() == settings.PRODUCTION_DOMAIN.replace("app.", ""):
-            # Redirect to user dashboard for logged in users
-            if request.user.is_authenticated:
-                return redirect("projects_dashboard")
-
-            # Redirect to ``about.`` in production
-            if not settings.DEBUG:
-                query_string = f"?ref={settings.PRODUCTION_DOMAIN}"
-                if request.META["QUERY_STRING"]:
-                    # Small hack to not append `&` to URLs without a query_string
-                    query_string += "&" + request.META["QUERY_STRING"]
-
-                # Do a 302 here so that it varies on logged in status
-                return redirect(f"https://about.readthedocs.com/{query_string}", permanent=False)
-
-        # Request hitting app.readthedocs.org / app.readthedocs.com
-        #
         # Redirect to user dashboard for logged in users
         if request.user.is_authenticated:
-            return redirect("projects_dashboard")
+            return redirect(reverse("projects_dashboard"))
 
-        # Redirect to login page
-        return redirect(reverse("account_login"))
+        # Redirect to ``about.`` in production
+        if not settings.DEBUG:
```

Comment:
Do we need to check for DEBUG? I think this page is isolated and can always run this logic.